### PR TITLE
Portaudio 19.7.0

### PR DESCRIPTION
**Pull request by prez**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [x] I generally don't use the affected packages but briefly tested this PR

Edit: removed the audacity and sndio patches as they (potentially) need to be updated.

### CODE REVIEW

**Comment by ericonr** [comment]
The sndio patch is still necessary (if we want to keep the build option), given that it hasn't been upstreamed yet. Don't think it should be a blocker, though.

The Audacity one is relevant, though. Would be important to run a test build.

Are you sure you need to revbump all these packages? The soname isn't changing.

**Comment by prez** [comment]
@ericonr 
> The sndio patch is still necessary (if we want to keep the build option), given that it hasn't been upstreamed yet. Don't think it should be a blocker, though.

Should I leave it as is or remove the build option then?

> The Audacity one is relevant, though. Would be important to run a test build.

Audacity fails to build without it on my machine due to a glib issue in the configure stage. Is there anything else I can do?

> Are you sure you need to revbump all these packages? The soname isn't changing.

I am not - I thought a rebuild would be necessary anyway. So nothing would need a revbump here?

File: srcpkgs/portaudio/template [hunk 1]
```diff
@@ -11,8 +11,8 @@ short_desc="Portable cross-platform audio I/O library"
 maintainer="Érico Nogueira <ericonr@disroot.org>"
 license="MIT"
 homepage="http://www.portaudio.com"
-distfiles="http://www.${pkgname}.com/archives/pa_stable_v${version%.*}_${version#*.}.tgz"
-checksum=f5a21d7dcd6ee84397446fa1fa1a0675bb2e8a4a6dceb4305a8404698d8d1513
+distfiles="http://files.portaudio.com/archives/pa_stable_v190700_20210406.tgz"
+checksum=47efbf42c77c19a05d22e627d42873e991ec0c1357219c0d74ce6a2948cb2def
```
Comment (Chocimier) [inline srcpkgs/portaudio/template:15]: Version scheme change is problematic. Now one has to update version *and* distfiles. 19.7.0 to 190600.20161030 is also seen downgrade by xbps, so `reverts` with old versions would need to be added, and xbps-still would list it as downgrade.

Comment (prez) [reply srcpkgs/portaudio/template]: I changed the source location to github and added a `reverts`.

**Comment by prez** [comment]
@ericonr 

Audacity builds and runs fine. 
Are you sure I don't need to revbump anything? Installing 19.7.0 without revbumping and updating xournalpp is impossible, for example.

File: srcpkgs/portaudio/template [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # Template file for 'portaudio'
 pkgname=portaudio
-version=190600.20161030
-revision=6
-wrksrc=portaudio
+reverts="190600.20161030_6"
```
Comment (sgn) [inline srcpkgs/portaudio/template:3]: `reverts="190600.20161030_1 190600.20161030_2 190600.20161030_3 190600.20161030_4 190600.20161030_5 190600.20161030_6"`

**Pull request closed without merging**